location/OBARegionHelper.m

```diff
@@ -63,8 +65,7 @@ - (void)setNearestRegion {
             OBARegionV2 *region = (OBARegionV2 *)obj;
             BOOL showExperimentalRegions = NO;
 
-            if ([[NSUserDefaults standardUserDefaults] boolForKey:@"kOBAShowExperimentalRegionsDefaultsKey"]) showExperimentalRegions = [[NSUserDefaults standardUserDefaults]
-                                                                                                                                         boolForKey:@"kOBAShowExperimentalRegionsDefaultsKey"];
+            if ([[NSUserDefaults standardUserDefaults] boolForKey:@"kOBAShowExperimentalRegionsDefaultsKey"]) showExperimentalRegions = [[NSUserDefaults standardUserDefaults] boolForKey:@"kOBAShowExperimentalRegionsDefaultsKey"];
```

Comment:
As long as you're in here, would you mind moving the string literal into a constant and using that constant everywhere in the codebase? String literals are way too easy to mess up and mistype.

in the .h file:

    extern NSString * const kOBAShowExperimentalRegionsDefaultsKey;

in the .m file:

    NSString * const kOBAShowExperimentalRegionsDefaultsKey = @"kOBAShowExperimentalRegionsDefaultsKey";